utils/node_local_test.py

```diff
@@ -1888,6 +1914,48 @@ def test_with_path(self):
         if dfuse.stop():
             self.fatal_errors = True
 
+    @needs_dfuse_no_cache
+    def test_file_nospace(self):
+        """Test for file size accounting a correct ENOSPACE handling"""
+
+        filename = os.path.join(self.dfuse.dir, 'new_file')
+
+        fd = open(filename, 'wb', buffering=0)
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations